ci/test/04_install.sh

```diff
@@ -33,18 +33,28 @@ if [ -z "$RUN_CI_ON_HOST" ]; then
   DOCKER_ID=$(docker run $DOCKER_ADMIN -idt --mount type=bind,src=$BASE_BUILD_DIR,dst=$BASE_BUILD_DIR --mount type=bind,src=$CCACHE_DIR,dst=$CCACHE_DIR -w $BASE_BUILD_DIR --env-file /tmp/env $DOCKER_NAME_TAG)
 
   DOCKER_EXEC () {
-    docker exec $DOCKER_ID bash -c "cd $PWD && $*"
+    docker exec $DOCKER_ID bash -c "export PATH=$BASE_SCRATCH_DIR/bins/:\$PATH && cd $PWD && $*"
   }
 else
   echo "Running on host system without docker wrapper"
   DOCKER_EXEC () {
-    bash -c "cd $PWD && $*"
+    bash -c "export PATH=$BASE_SCRATCH_DIR/bins/:\$PATH && cd $PWD && $*"
   }
 fi
 
 DOCKER_EXEC free -m -h
-DOCKER_EXEC echo "Number of CPUs \(nproc\): $(nproc)"
+DOCKER_EXEC echo "Number of CPUs \(nproc\):" \$\(nproc\)
 
 ${CI_RETRY_EXE} DOCKER_EXEC apt-get update
 ${CI_RETRY_EXE} DOCKER_EXEC apt-get install --no-install-recommends --no-upgrade -qq $PACKAGES $DOCKER_PACKAGES
 
+if [ "$USE_BUSY_BOX" = "true" ]; then
+  echo "Setup to use BusyBox utils"
+  DOCKER_EXEC mkdir -p $BASE_SCRATCH_DIR/bins/
+  # tar excluded for now because it requires passing in the exact archive type in ./depends (fixed in later BusyBox version)
+  # find excluded for now because it does not recognize the -delete option in ./depends (fixed in later BusyBox version)
```

Comment:
Support for `-delete` was added to `find` in BusyBox `1.6.0` (2007). The [following](https://github.com/bitcoin/bitcoin/blob/master/depends/packages/xcb_proto.mk#L25) inside an Alpine container (BusyBox v1.30.1) seems to work ok:
```bash
touch something.pyo something.pyc
find -name "*.pyc" -delete && find -name "*.pyo" -delete
```

What error were you seeing?

Reply:
2007 oh my, please don't tell me anything that old is being used :fearful:

Reply:
I have no idea what bionic is doing, but https://github.com/bitcoin/bitcoin/pull/17011#issuecomment-537507555